plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImageWizardModel.java

```diff
@@ -268,7 +268,15 @@ public void setImageName(final String imageName) {
 		final DockerImageURI uri = new DockerImageURI(imageName);
 		setResourceName(dockerImage2OpenshiftResourceConverter.convert(uri));
 	}
-	
+
+	@Override
+	public void setImageName(String imageName, boolean forceUpdate) {
+		if(forceUpdate && this.imageName != null && this.imageName.equals(imageName)) {
```

Comment:
I have tried it, and it did not work. The only combination of calls that works for me, is this. The use case itself is rare - select, clear input field, select the same again, and only in this case that additional firing is done.